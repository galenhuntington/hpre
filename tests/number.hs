number :: Double
number = 299_792_458.123_456
