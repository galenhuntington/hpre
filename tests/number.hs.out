number :: Double
number = 299792458.123456

